Shippo/Shipment.cs

```diff
@@ -1,18 +1,20 @@
 ﻿using System;
+using System.Collections.Generic;
```

Comment:
New typed attributes were added and old ones only reordered to match API doc ordering